op-deployer/pkg/deployer/manage/add_game_type_test.go

```diff
@@ -0,0 +1,64 @@
+package manage
+
+import (
+	"context"
+	"log/slog"
+	"math/big"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/op-deployer/pkg/deployer/opcm"
+	"github.com/ethereum-optimism/optimism/op-deployer/pkg/deployer/testutil"
+	"github.com/ethereum-optimism/optimism/op-service/testlog"
+	"github.com/ethereum-optimism/optimism/op-service/testutils"
+	"github.com/ethereum-optimism/superchain-registry/validation"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAddGameType(t *testing.T) {
+	rpcURL := os.Getenv("SEPOLIA_RPC_URL")
+	require.NotEmpty(t, rpcURL, "must specify RPC url via SEPOLIA_RPC_URL env var")
+
+	afacts, _ := testutil.LocalArtifacts(t)
+	v200SepoliaAddrs := validation.StandardVersionsSepolia["op-contracts/v2.0.0-rc.1"]
+	testCacheDir := testutils.IsolatedTestDirWithAutoCleanup(t)
+
+	cfg := AddGameTypeConfig{
+		L1RPCUrl:         rpcURL,
+		Logger:           testlog.Logger(t, slog.LevelInfo),
+		ArtifactsLocator: afacts,
+		Input: opcm.AddGameTypeInput{
+			SaltMixer: "foo",
+			// The values below were pulled from the Superchain Registry.
```

Comment:
Maybe update this comment or add another one (or update test name) to make it clear you are testing against op-sepolia here